src/types_test.ts

```diff
@@ -94,10 +94,6 @@ describeMathCPU('ConcreteTensor Test', () => {
   it('Read value', () => {
     const v1 = new CT(scalar(10), 'foo');
     expect(v1.value().dataSync()).toEqual(new Float32Array([10]));
-
-    let v1Value = v1.value();
```

Comment:
I don't quite understand the original intent of these deleted lines.  Please let me know if there's something that should actually be tested here.